eng/pipelines/partner-release.yml

```diff
@@ -25,41 +25,53 @@ resources:
 variables:
   BuildToolScripts: $(Pipeline.Workspace)/azure-sdk-build-tools/scripts
   Artifacts: $(Pipeline.Workspace)/artifacts
+  SignedArtifact: $(Pipeline.Workspace)/packages-signed
 
-pool:
-  vmImage: 'windows-2019'
+jobs:
+- job: Signing
+  pool:
+    name: azsdk-pool-mms-win-2019-general
+    vmImage: windows-2019
+  steps:
+  - checkout: azure-sdk-build-tools
+    path: azure-sdk-build-tools
 
-steps:
-- checkout: self
-  path: azure-sdk-for-java
+  - task: PowerShell@2
+    displayName: 'Download packages from blob storage'
+    inputs:
+      targetType: filePath
+      filePath: '$(BuildToolScripts)/copy-from-azuresdkpartnerdrops.ps1'
+      arguments: '$(Artifacts) ${{ parameters.BlobPath }} $(azuresdkpartnerdrops-access-key)'
 
-- checkout: azure-sdk-build-tools
-  path: azure-sdk-build-tools
+  - template: tools/java-esrp-signing/java-esrp-signing.yml@azure-sdk-build-tools
+    parameters:
+      ArtifactDirectory: $(Artifacts)
 
-- task: PowerShell@2
-  displayName: 'Download packages from blob storage'
-  inputs:
-    targetType: filePath
-    filePath: '$(BuildToolScripts)/copy-from-azuresdkpartnerdrops.ps1'
-    arguments: '$(Artifacts) ${{ parameters.BlobPath }} $(azuresdkpartnerdrops-access-key)'
-
-- template: tools/java-esrp-signing/java-esrp-signing.yml@azure-sdk-build-tools
-  parameters:
-    ArtifactDirectory: $(Artifacts)
-
-- publish: $(Artifacts)
-  artifact: packages-signed
-
-- template: tools/gpg/gpg.yml@azure-sdk-build-tools
-- template: /eng/pipelines/templates/steps/java-publishing.yml
-  parameters:
-    ArtifactDirectory: $(Artifacts)
-    Target: MavenCentral
-    BuildToolsPath: $(Pipeline.Workspace)/azure-sdk-build-tools
-    JavaRepoRoot: $(Pipeline.Workspace)/azure-sdk-for-java
-    ShouldPublish: ${{ parameters.ShouldPublish }}
-    StageOnly: ${{ parameters.StageOnly }}
-- template: /eng/common/pipelines/templates/steps/publish-artifact.yml
-  parameters:
-    ArtifactName: packages-mavencentral-$(System.JobAttempt)
-    ArtifactPath: $(Artifacts)
\ No newline at end of file
+  - publish: $(Artifacts)
+    artifact: packages-signed
+- job: Release
+  dependsOn: Signing
+  pool:
+    name: azsdk-pool-mms-win-2019-general
+    vmImage: windows-2019
+  steps:
+  - checkout: self
+    path: azure-sdk-for-java
```

Comment:
It may require some testing, but if you want to speed up the partner release pipeline by several minutes, you could copy over the sparse checkout logic we use for the normal release pipeline: https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/stages/archetype-java-release.yml#L166-L174

Reply:
We should do that.   I'll follow with another PR